api/openapi-spec/swagger.json

```diff
@@ -19195,7 +19195,7 @@
   },
   "info": {
     "title": "Kubernetes",
-    "version": "v2.17.1"
```

Comment:
guess it is a bug in "make update". We need to revert this unintentional change.

Reply:
Not sure if it's a bug in make update or something intentional( so that we need to manually change version number?) Don't know the right behavior here. I will revert this and I created an issue #132 to track.